components/infobox/wikis/brawlhalla/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,131 @@
+---
+-- @Liquipedia
+-- wiki=brawlhalla
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Player = require('Module:Infobox/Person')
+local Class = require('Module:Class')
+local Variables = require('Module:Variables')
+local Math = require('Module:Math')
+local ActiveYears = require('Module:YearsActive')
+local PlayersSignatureLegends = require('Module:PlayersSignatureLegends')
+
+local _PAGENAME = mw.title.getCurrentTitle().prefixedText
```

Comment:
if `self.pagename` exists, why is this one needed?

Reply:
`self.pagename = mw.title.getCurrentTitle().text = {{PAGENAME}}`
`_PAGENAME` here is `{{FULLPAGENAME}}`
so technically they are different

i will test if we can use `self.pagename` instead

Reply:
okay we can not use `self.pagename` in the functions where `_PAGENAME` is used as it is part of the `CustomInjector` class and hence does not have self.pagename set there
we can use `_player.pagename` instead